### PR TITLE
add example yaml for NG Monitoring

### PR DESCRIPTION
**Pull request by KanShiori**
### Code changes

- [ ] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [ ] Manual test 
- [x] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

File: examples/tidb-ng-monitoring/tngm.yaml [hunk 1]
```diff
@@ -0,0 +1,182 @@
+apiVersion: pingcap.com/v1alpha1
```
Comment (DanielZhangQD) [inline examples/tidb-ng-monitoring/tngm.yaml:1]: What about moving this file to the `advanced` dir and only putting a basic example here?

File: examples/tidb-ng-monitoring/tngm.yaml [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/overview/#container-images
+  # imagePullPolicy: IfNotPresent
+
+  ## If private registry is used, imagePullSecrets may be set
+  ## You can also set this in service account
+  ## Ref: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+  # imagePullSecrets:
+  # - name: secretName
+
+  ## Specify pod priorities of pods in TidbCluster, default to empty.
```
Comment (csuzhangxc) [inline examples/tidb-ng-monitoring/tngm.yaml:38]: not `TidbCluster`

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: examples/advanced/tidb-ng-monitoring.yaml [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    ##   - TZ
+    ##   - SERVICE_NAME
+    ##   - PEER_SERVICE_NAME
+    ##   - HEADLESS_SERVICE_NAME
+    ##   - SET_NAME
+    ##   - HOSTNAME
+    ##   - CLUSTER_NAME
+    ##   - POD_NAME
+    ##   - BINLOG_ENABLED
+    ##   - SLOW_LOG_FILE
```
Comment (csuzhangxc) [inline examples/advanced/tidb-ng-monitoring.yaml:104]: these two lines can be dropped?

File: examples/advanced/tidb-ng-monitoring.yaml [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+    #       fieldPath: status.myEnv2
+
+    ## NodeSelector of ng monitoring pods
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+    # nodeSelector:
+    #   node-role.kubernetes.io/ngm: true
+
+    ## Tolerations are applied to ng monitoring pods, allowing (but do not require) pods to be scheduled onto nodes with matching taints.
+    ## This cluster-level `tolerations` only takes effect when no component-level `tolerations` are set.
+    ## e.g. if `pd.tolerations` is not empty, `tolerations` here will be ignored.
```
Comment (csuzhangxc) [inline examples/advanced/tidb-ng-monitoring.yaml:121]: `pd.tolerations`

File: examples/advanced/tidb-ng-monitoring.yaml (hunk 3 above)
Comment (DanielZhangQD) [reply examples/advanced/tidb-ng-monitoring.yaml:104]: Please also check if the others are set for ngm.

File: examples/advanced/tidb-ng-monitoring.yaml (hunk 4 above)
Comment (KanShiori) [reply examples/advanced/tidb-ng-monitoring.yaml]: done

File: examples/advanced/tidb-ng-monitoring.yaml (hunk 3 above)
Comment (KanShiori) [reply examples/advanced/tidb-ng-monitoring.yaml]: done

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**